packages/compiler-sfc/src/script/defineProps.ts

```diff
@@ -260,9 +260,15 @@ function genRuntimePropFromType(
         // prop has corresponding static default value
         defaultString = `default: ${ctx.getString(prop.value)}`
       } else {
+        let paramsString = ''
+        if (prop.params.length) {
+          const start = prop.params[0].start
```

Comment:
due to `default` only accepts one parameter, it might be:
```js
const paramName = prop.params.length?prop.params[0].name:''
```

Reply:
At first, I thought the same way, but considering the possibility of the following scenarios:
```js
foo(a, b) {
    console.log(b);
    return 0;
}
```
If the user accesses other parameters, it may throw a runtime error. If there happens to be a variable with the same name in the outer scope, that variable will be accessed.

If the user uses function declaration or arrow function, such issues will not occur.

In addition, we need to consider that if users define parameters using the `ObjectPattern` or `RestElement` format, the parameter name cannot be accessed using `prop.params[0].name`.

Reply:
Okay, sounds good.